docs/articles/documentation/guides/basic-guides/obtain-client-side-info.md

```diff
@@ -70,6 +70,27 @@ test('My Test', async t => {
 });
 ```
 
+### Pass Parameters to Client Functions
+
+This example shows the `getLocationPart` client function that returns a [Location](https://developer.mozilla.org/en-US/docs/Web/API/Location) property value. The property name is passed in the `locationPart` parameter.
```

Comment:
I meet the `passed as parameter` form more often.
Can we change to `The property name is passed as the `locationPart` parameter.`?